bin/woff2_compress.js

```diff
@@ -18,22 +19,28 @@ var parser = new argparse.ArgumentParser({
 });
 
 parser.addArgument([ 'infile' ],  { nargs: 1, help: 'Input file' });
-parser.addArgument([ 'outfile' ], { nargs: 1, help: 'Output file' });
+parser.addArgument([ 'outfile' ], { nargs: '*', help: 'Output file (- for stdout)' });
 
 ////////////////////////////////////////////////////////////////////////////////
 
 let args = parser.parseArgs();
+const [ infile ] = args.infile;
```

Comment:
too strange improvement for `args.infile[0]`

Reply:
No problem, I'll revert it back to the original form.